csrc/utils.cpp

```diff
@@ -37,86 +37,47 @@ c10::ThreadPool* getThreadPool() {
   return &pool;
 }
 
-C10_DIAGNOSTIC_PUSH_AND_IGNORED_IF_DEFINED("-Wunused-function")
-void debugPrint(const c10::TensorTypePtr& type) {
```

Comment:
@jjsjann123 do you still use this? I wanted an abbreviated version which I added below.

Reply:
no need for these any more, since it's now easy to switch to a debug build with gdb.

Reply:
Also, `TensorType` is a TorchScript thing that's obsolete in our code base now. The removal looks safe to me.